package.json

```diff
@@ -5,6 +5,9 @@
   "author": "Krishang <krishang@thirdweb.com>",
   "license": "Apache-2.0",
   "devDependencies": {},
-  "dependencies": {},
+  "dependencies": {
+    "@account-abstraction/contracts": "^0.7.0",
+    "@openzeppelin/contracts": "^5.0.0"
+  },
```

Comment:
Is it possible to use `forge install` so we only use 1 package manager instead of 2?

Reply:
difficult in this case

Reply:
ok we're good now